### PR TITLE
Describe in more detail how to set up C# tests in Xamarin on Mac

### PR DESCRIPTION
**Pull request by Malwine**
Dear Maintainers,
I started exercising C# with exercism and struggled setting up the tests in Xamarin on Mac. Once I figured it out I thought about contributing the process to the `Running the tests` site.
I put in some screenshots and described important steps.

Let me know if that is a useful contribution.

Moreover, I am wondering if it is ok that I used relative links to the pictures?

Hope to hear from you :)
Malwine

### CODE REVIEW

**Comment by ErikSchierboom** [comment]
Thanks a lot for this work! We really appreciate it. The image links should be absolute links, as the images themselves are not copied to the website. In the current version, the image link is:

<pre>
-Xamarin NUnit
</pre>

Could you prefix the relative image paths with `http://x.exercism.io/v3/tracks/csharp/docs/`? After that, we'll merge this!

**Comment by Malwine** [comment]
@ErikSchierboom Sure, I'll prefix the picture links asap. 

Another question is why the Travis build is red. I am not sure if it has to do with my changes?

**Comment by ErikSchierboom** [comment]
@Malwine No that has nothing to do with you. I'll be fixing that! Don't worry about it.

File: docs/TESTS.md [hunk 1]
```diff
@@ -20,9 +20,27 @@ Once you have been able to compile the code it will create a DLL in the ```\bin\
 The NUnit runner will automatically reload the DLL if it has been updated.
 
 ### Mac
-Xamarin Studio also ships with NUnit. From the new project dialog, just select an NUnit class library.
+Xamarin Studio also ships with NUnit. To set the tests up you will have to add NUnit as a new project to your solution, name it correctly and set a reference to your solution.
```
Comment (ErikSchierboom) [inline docs/TESTS.md]: Perhaps "add NUnit as a new project to your solution" could be rephrased to something like " add an NUnit library project to your solution"? As you are not actually adding NUnit as a new project, but just use the template.

Comment (Malwine) [inline docs/TESTS.md]: Yeah, sure. The clearer the better!

**Comment by ErikSchierboom** [comment]
@Malwine This has been merged! Thanks a lot for the contribution. :tada: :+1:

**Pull request merged**